docker-compose.dev.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.7"
+
+# development docker-compose setup overrides for Retrospring
+# debugging support using rdebug-ide and preconfigured DB credentials
+
+services:
+  web:
```

Comment:
since it was missing from the "prod" and the previous docker-compose yamls, how do you deal with starting up Sidekiq locally?